packages/brick_offline_first/lib/src/offline_first_repository.dart

```diff
@@ -176,7 +176,9 @@ abstract class OfflineFirstRepository<TRepositoryModel extends OfflineFirstModel
   /// is ignorable (e.g. eager loading). Defaults to `false`.
   @override
   Future<List<TModel>> get<TModel extends TRepositoryModel>({
-    OfflineFirstGetPolicy policy = OfflineFirstGetPolicy.awaitRemoteWhenNoneExist,
+    OfflineFirstGetPolicy policy =
+        OfflineFirstGetPolicy.awaitRemoteWhenNoneExist,
+    OfflineFirstGetPolicy? policyForAssociations,
```

Comment:
Semantically, I think this makes more sense as `associationPolicy`, but I'm open to hearing your rationale behind `policyForAssociation`

```suggestion
    OfflineFirstGetPolicy? associationPolicy,
```

Reply:
If you like `associationPolicy`, the argument would be alphabetized first